components/unione/actions/send-email/send-email.mjs

```diff
@@ -0,0 +1,227 @@
+      options: SKIP_UNSUBSCRIBE_OPTIONS,
+      optional: true,
+    },
+    globalLanguage: {
+      type: "string",
+      label: "Global Language",
+      description: "The language of the unsubscribe footer and unsubscribe page.",
+      options: GLOBAL_LANGUAGE_OPTIONS,
+      optional: true,
+    },
+    templateEngine: {
+      type: "string",
+      label: "Template Engine",
+      description: "The [template engine](https://docs.unione.io/en/template-engines) for handling the substitutions(merge tags).",
+      optional: true,
+      options: TEMPLATE_ENGINE_OPTIONS,
+    },
+    globalSubstitutions: {
+      type: "object",
+      label: "Global Substitutions",
+      description: "Object for passing the substitutions(merge tags) common for all recipients - e.g., company name. If the substitution names are duplicated in recipient 'substitutions', the values of the variables will be taken from the recipient 'substitutions'. Example: { \"body\": { \"html\": \"Hello, {name}!\", \"plaintext\": \"Hello, {name}!\", \"amp\": \"Hello, {name}!\"}, \"subject\": \"Hello, {name}!\", \"from_name\": \"John Doe\", \"options\": { \"unsubscribe_url\": \"https://example.com/unsubscribe\" } }.",
+      optional: true,
+    },
+    globalMetadata: {
+      type: "object",
+      label: "Global Metadata",
+      description: "Object for passing the metadata common for all the recipients, such as 'key': 'value'. Max key quantity: 10. Max key length: 64 symbols. Max value length: 1024 symbols.",
+      optional: true,
+    },
+    body: {
+      type: "object",
+      label: "Body",
+      description: "Contains HTML/plaintext/AMP parts of the email. Either html or plaintext part is required. Example: { \"html\": \"Hello, {name}!\", \"plaintext\": \"Hello, {name}!\", \"amp\": \"Hello, {name}!\" }.",
+    },
+    subject: {
+      type: "string",
+      label: "Subject",
+      description: "Email subject",
+    },
+    fromEmail: {
+      type: "string",
+      label: "From Email",
+      description: "Sender's email. Required only if `Template ID` prop is empty.",
+      optional: true,
+    },
+    fromName: {
+      type: "string",
+      label: "From Name",
+      description: "Sender's name",
+      optional: true,
+    },
+    replyTo: {
+      type: "string",
+      label: "Reply To",
+      description: "Reply-to email (in case it's different to sender's email)",
+      optional: true,
+    },
+    replyToName: {
+      type: "string",
+      label: "Reply To Name",
+      description: "Reply-To name (if `Reply To` email is specified and you want to display not only this email but also the name)",
+      optional: true,
+    },
+    trackLinks: {
+      type: "string",
+      label: "Track Links",
+      description: "1 = click tracking is on (default), 0 = click tracking is off. To use track_links = 0, you need to ask UniOne support to enable this feature.",
+      options: TRACK_OPTIONS,
```

Comment:
Please adjust the props that use `TRACK_OPTIONS` options to be of the `boolean` type instead, the run method can convert the boolean to the format expected by the API.